src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/Controllers/Domain/PlayersController.cs

```diff
@@ -58,13 +60,15 @@ public class PlayersController : Controller
             _requestService = requestService;
         }
 
+#pragma warning disable S2360 // Optional parameters should not be used
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not suppress issues. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1309)